charmcraft.yaml

```diff
@@ -43,6 +23,11 @@ bases:
       channel: "22.04"
 
 
+# Metadata
+peers:
+  peer:
+    interface: airbyte
```

Comment:
question: What is the purpose of this peer relation?

I saw it in the other charm as well. If it is meant to be between the UI and the server, is it supposed to be a peer relation?

Reply:
The peer relation is the relation of the charm with itself, it provides a relation databag that allows us to store relevant data to be accessed by the application's units later on (e.g. db connection).

Reply:
I see. I was confused by both relations being named `airbyte`.